### ISSUE TITLE
Shields can block stuns from melee weapons

### ISSUE THREAD

**Issue by Hackole** [HippieStationArchive/HippieStation13-2#203]
Whilst this was (technically) a bug from the old code, it was good balance that you could get past any kind of shield if you had a melee weapon that stunned - telebatons, stunprods, batons, etc.

**Comment by KittyKathy** [comment]
Pretty sure this is intended as in tg added this. Things that can block stuff as in double eswords can now block melee attacks that includes stunbatons. It was pretty stupid that shields can block a fire axe attack on you but one telescopic baton is like lol nope. All that did was make the stun meta worse. Should keep it like this that was not good balance. Anything that is a guaranteed gg no re without a counter is not good balance one baton that is unblockable with a shield is not good balance. If they get a single hit in they stun lock you and beat you at the same time it's called harm baton and it sucks.

**Comment by Hackole** [comment]
I mean, if you want literally everyone to carry around a riotshield, this is you accomplish that - by making them block EVERYTHING.

**Comment by ghost** [comment]
I agree with @Hackole melee weapons should block ranged weaponary/throwing shit, not stuns

**Comment by KittyKathy** [comment]
Not everyone has access to shields/doubleeswords, one is a security thing in the armory and only other place to get it is cargo ordering it the other is a traitor item. I'm just saying batons should not be 100% chance as it's just going to make the meta where people will just charge at people who have double eswords and shields with batons because they know all they need to do is land one hit and then it's pretty much over.

**Comment by Hackole** [comment]
Hence why when using a shield you would keep a distance to people, and when using a double esword, you wouldn't just deploy it then run around ONLY using that.
